test/services/jobs/return-logs/fetch-return-logs.service.test.js

```diff
@@ -47,6 +49,10 @@ describe('Fetch return logs service', () => {
     it('should return one return log payload', async () => {
       const result = await FetchReturnLogsService.go(false, licence.licenceRef)
 
+      const primaryPurpose = await PrimaryPurposeModel.query().findById(returnRequirementPurpose.primaryPurposeId)
+      const secondaryPurpose = await SecondaryPurposeModel.query().findById(returnRequirementPurpose.secondaryPurposeId)
+      const tertiaryPurpose = await PurposeModel.query().findById(returnRequirementPurpose.purposeId)
+
```

Comment:
These records are pre-seeded in the DB.

We should be able to use their helper `select()` functions to pick them, then assign those picked ones to your test return requirement purposes (just as done with `Region.select()`).

You can then use them again to assert the legacy IDs and descriptions are what you expected.